galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/dependencies/impl/HttpRequestFactory.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.ras.couchdb.internal.dependencies.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.http.HttpRequest;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpHead;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+
+import dev.galasa.framework.spi.Environment;
+
+public class HttpRequestFactory {
+
+    private Map<String, String> headers = new HashMap<String,String>();
+
+    public HttpRequestFactory(Environment environment) {
+        headers.put("Accept", "application/json");
+        headers.put("Content-Type", "application/json");
+        headers.put( "Authorization", environment.getenv("GALASA_RAS_TOKEN"));
```

Comment:
Might be worth putting `GALASA_RAS_TOKEN` in a constant rather than hardcoding it here to make it easier to find in the future